### DISCUSSION TITLE
[get-folder-size] doesn't export its types

### DISCUSSION THREAD

**Discussion by jsoref** [DefinitelyTyped/DefinitelyTyped#68634] [Issues with a @types package]
This declares a namespace:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/451dc8fc19383bc12af59522020e571957f1684e/types/get-folder-size/index.d.ts#L5
This declares a function that returns a type from that namespace:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/451dc8fc19383bc12af59522020e571957f1684e/types/get-folder-size/index.d.ts#L73-L76
This makes that function the only export (but doesn't export the types needed for arguments/returns):
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/451dc8fc19383bc12af59522020e571957f1684e/types/get-folder-size/index.d.ts#L78

I'm not claiming this is a correct patch, but with it, I can actually use this type to do things:

```diff
diff --git a/node_modules/@types/get-folder-size/index.d.ts b/node_modules/@types/get-folder-size/index.d.ts
index a5fc48261..70dc2634d 100644
--- a/node_modules/@types/get-folder-size/index.d.ts
+++ b/node_modules/@types/get-folder-size/index.d.ts
@@ -2,8 +2,7 @@
 
 import { PathLike } from "fs";
 
-export as namespace getFolderSize;
-declare namespace getFolderSize {
+export namespace getFolderSizeNS {
     /**
      * Returns the size of the folder. If any errors are encountered while traversing the folder, they are silently ignored.
      *
@@ -70,11 +69,11 @@ declare namespace getFolderSize {
  *
  * @async
  */
-function getFolderSize(
+export function getFolderSize(
     itemPath: PathLike,
-    options?: getFolderSize.Options,
-): Promise<getFolderSize.FolderSizeInfo>;
+    options?: getFolderSizeNS.Options,
+): Promise<getFolderSizeNS.FolderSizeInfo>;
 
 }
 
-export default getFolderSize;
+export default getFolderSizeNS.getFolderSize;
```

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
Hi, those types are exported, but in rather clunky way:
```ts
const options: getFolderSize.Options = {};
```
(so through namespace like accessor, similar to NodeJS.{T}). Types can be improved for sure
